src/cplscheme/CouplingData.hpp

```diff
@@ -103,7 +108,7 @@ class CouplingData {
   mesh::PtrData _data;
 
   /// Sample values of previous iteration (end of time window).
-  time::Sample _previousIteration;
+  time::Storage _timeStepsStoragePrevious;
```

Comment:
```suggestion
  time::Storage _previousTimeStepsStorage;
```
? Could be more consistent with the naming above

Reply:
Done. [72c63a2](https://github.com/precice/precice/pull/1794/commits/72c63a2b096021f4db7ea6b9a2eb0c42d7f71336)